kubernetes/cmsweb/ingress/ing-dmwm.yaml

```diff
@@ -80,3 +80,10 @@ spec:
             name: ms-unmer-t2t3
             port:
               number: 8242
+      - path: /ms-pileup/data
```

Comment:
should we simply use the same pattern used for the other WMCore services (different than MSUnmerged)? Thus, defining this path is `/ms-pileup` only.

Reply:
Hi @amaltaro, thanks for the review. I have updated the path.